pkg/api/v1beta1/dynakube/activegate_types.go

```diff
@@ -62,7 +62,7 @@ type ActiveGateSpec struct {
 	// server.p12: certificate+key pair in pkcs12 format
 	// password: passphrase to read server.p12
 	// +optional
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="TlsSecretName",order=10,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced","urn:alm:descriptor:com.tectonic.ui:text"}
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="TLSSecretName",order=10,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced","urn:alm:descriptor:com.tectonic.ui:text"}
 	TlsSecretName string `json:"tlsSecretName,omitempty"`
```

Comment:
I wouldn't mess with `displayName="TLSSecretName"`
but `TlsSecretName string` can be changed

Reply:
and keep `json:"tlsSecretName,omitempty"` so it keeps old name

Reply:
Done in b58dca7f